### PR TITLE
fix(header): 11851-remove-chat-with-us-from-header

### PR DESCRIPTION
**Pull request by vkozio**
[11851](https://kontur.fibery.io/Tasks/Task/Remove-Chat-with-us-from-header-11851)

image

### CODE REVIEW

File: src/RoutesWrap.tsx [hunk 1]
```diff
@@ -110,7 +110,6 @@ export const CommonRoutesFeatures = ({
       <Suspense fallback={null}>
         {userModel.hasFeature(AppFeature.HEADER) && (
           <AppHeader
```
Comment (propakov) [inline src/RoutesWrap.tsx:112]: pls update ui-kit
https://github.com/konturio/ui/pull/32/files#r983295199

File: src/RoutesWrap.tsx [hunk 2]
```diff
@@ -110,7 +110,6 @@ export const CommonRoutesFeatures = ({
...
-            chatBtnText={i18n.t('app_header.title')}
```
Comment (propakov) [inline src/RoutesWrap.tsx:113]: pls remove app_header.title from i18n json files

**Review by propakov — CHANGES_REQUESTED** [review]

File: src/RoutesWrap.tsx (hunk 2 above)
Comment (vkozio) [reply src/RoutesWrap.tsx:113]: done

File: src/RoutesWrap.tsx (hunk 1 above)
Comment (vkozio) [reply src/RoutesWrap.tsx:112]: done

**Review by propakov — APPROVED** [review]
👍

**Pull request merged**